### ISSUE TITLE
HTML body font (Source Sans Pro) isn't imported anymore: maybe warn the user or embed in the layout?

### ISSUE THREAD

**Issue by gremo** [sonata-project/SonataAdminBundle#7138]
AdminLT was updated by @core23 with https://github.com/sonata-project/SonataAdminBundle/pull/7032 from 2.3.8 to 2.4.15. The new version [removes the import rule](https://github.com/ColorlibHQ/AdminLTE/commit/235481d1d6324c1d4257f86c621700aa7f4a0b7f#diff-d01f962a85781ac8043e23efc98953b32b7d3358080a27b43d1d17af5dc7dd72), so right now the admin panel has a wrong font.

Should the developer import it overriding the layout?

```
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,300italic,400italic,600italic">
```

**Comment by VincentLanglet** [comment]
Can you try this PR https://github.com/sonata-project/SonataAdminBundle/pull/7132 on your project @gremo ?
It's supposed to fix the issue.

**Comment by VincentLanglet** [comment]
Will be fixed in 3.98.2
